testsuite/python/lb_shear.py

```diff
@@ -31,7 +31,7 @@
 # Box size will be H +2 AGRID to make room for walls.
 # The number of grid cells should be divisible by four and 3 in all directions
 # for testing on multiple mpi nodes.
-H = 12 * AGRID
+H = 10 * AGRID
```

Comment:
Why change the box height?

Reply:
In the present test script, we don't need this modification anymore, so I am going to put it back.